pom.xml

```diff
@@ -19,8 +24,9 @@
   </licenses>
 
   <scm>
-    <url>https://github.com/rcarz/jira-client</url>
-    <connection>https://github.com/rcarz/jira-client</connection>
+    <url>https://github.com/mainstreethub/jira-client</url>
```

Comment:
The SCM section is not needed with the way our CI for library builds works.  It doesn't use the maven-release-plugin which is what reads this section of the POM.